modules/nw-scaling-ingress-controller.adoc

```diff
@@ -0,0 +1,93 @@
+// Module filename: nw-scaling-ingress-controller.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-ingress-controller-configuration-{context}"]
+= Scaling an ingress controller
+
+An ingress controller can be manually scaled up or down based on
+requirements such as the need to increase throughput. `oc`
+commands are used to scale the `IngressController` resource.
+
+.Procedure
+. View the current number of replicas of the ingress controller resource.
++
+----
+$ name="foo"  <1>
+$ oc get ingresscontroller/$name -n openshift-ingress-operator -o yaml
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  creationTimestamp: 2019-04-09T16:11:52Z
+  finalizers:
+  - ingress.openshift.io/ingress-controller
+  generation: 2
+  name: $name
+  namespace: openshift-ingress-operator
+  resourceVersion: "1187245"
+  selfLink: /apis/operator.openshift.io/v1/namespaces/openshift-ingress-operator/ingresscontrollers/$name
+  uid: 303a2234-5ae2-11e9-a3f2-0adb90d769be
+spec:
+  replicas: 2
+status:
+  availableReplicas: 2
+  conditions:
+  - lastTransitionTime: 2019-04-09T16:12:27Z
+    status: "True"
+    type: Available
+  domain: apps.example.com
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+  selector: ingress.operator.openshift.io/ingress-controller-deployment=$name
+----
+<1> Replace "foo" with the name of the ingress controller you would like to scale.
+
+The `replicas` field represents the desired number of pods, while `availableReplicas`
+represents the number of a running pods. If an ingress controller is created without
+specifying `replicas`, the field will not appear in the output and a default value
+of `2` will be used.
+
+. Scale the ingress controller to the desired number of replicas using
+the `oc patch` command.
++
+----
+$ count="3"  <2>
+$ oc patch -n openshift-ingress-operator ingresscontroller/$name \
+--patch '{"spec":{"replicas": $count}}' --type=merge
+----
+<2> Replace "3" with the number of desired ingress controller replicas.
```

Comment:
You probably see where I'm going with this, but an alternative would be:

```
$ oc patch -n openshift-ingress-operator ingresscontroller/default \
	--patch '{"spec":{"replicas": 3}}' --type=merge
```

Another advantage is that it's a copy/pastable one-liner

Reply:
Thanks, I agree. Removed.